lib/publiccloud/instance.pm

```diff
@@ -617,7 +617,7 @@ sub measure_boottime() {
     my ($self, $instance, $type) = @_;
     my $data_collect = get_var('PUBLIC_CLOUD_PERF_COLLECT', 1);
 
-    return 0 unless ($data_collect);
+    return 0 if (!$data_collect || is_openstack);
```

Comment:
This line here is not needed, because routine will be never called if openstack, being alreay skipped in crate_instances.

Reply:
That test should NOT have entered that block being is_openstack true: error is elsewhere.

Reply:
Point clarified in a huddle, resolving